src/components/StartScreen.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Grid from '@mui/material/Unstable_Grid2';
+import { Button } from '@mui/material';
+import StartLogo from '../assets/images/start_logo.png';
+
+const StartScreen = () => (
+  <>
+    {/* <Grid container height="100%" direction="column" justifyContent="center"> */}
```

Comment:
Not a big deal for the time being but we should remove this

Reply:
BE GONE COMMENT